src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
Basically we are misusing the Mixer as a global Mixer when it was intended to be a per-character mixer with support for extra synchronized objects with that character.

Softimage has a per-character mixer that supports clips: http://softimage.wiki.softimage.com/xsidocs/animmixer1.htm

Unity3D has a per-character mixer that supports clips: http://docs.unity3d.com/ScriptReference/Animation.html

Maya has a per character mixer: http://help.autodesk.com/view/MAYAUL/2016/ENU/?guid=GUID-B90726CA-6C86-4E9B-80F3-1F65745C3B3F

UE4 has a per-character animation Blueprint and a per-character AnimGraph: https://docs.unrealengine.com/latest/INT/Engine/Animation/AnimBlueprints/

Basically we are being non-standard in our usage of a Global Mixer and it is going to lead to continued issues because it is not the appropriate abstraction to the problem.